### PR TITLE
chore: disableLog default to true

### PR DESCRIPTION
**Pull request by boiawang**

### CODE REVIEW

File: packages/stylesheet-loader/src/index.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ module.exports = function(source) {
   }
 
   const parsedQuery = loaderUtils.parseQuery(this.query);
+
+  if (typeof parsedQuery.disableLog === 'undefined') {
+    parsedQuery.disableLog = true;
```
Comment (yuanyan) [inline packages/stylesheet-loader/src/index.js]: disableLog -> log，外部暴露可以简单化些

File: packages/jsx-compiler/src/modules/css.js [hunk 2]
```diff
@@ -78,6 +78,6 @@ function convertCSSUnit(raw, originExt = 'rem', targetExt = 'rpx') {
 }
 
 function createCSSModule(content) {
-  const loaderContext = { query: '?disableLog=true' };
+  const loaderContext = { query: '?log=true' };
```
Comment (wssgcg1213) [inline packages/jsx-compiler/src/modules/css.js]: log=false

File: packages/stylesheet-loader/src/Validation.js [hunk 3]
```diff
@@ -10,8 +10,8 @@ import particular from './particular';
 import chalk from 'chalk';
 
 class Validation {
-  static validate(camelCaseProperty, prop, value, selectors = '', position = {}, disableLog) {
-    if (disableLog) return {};
+  static validate(camelCaseProperty, prop, value, selectors = '', position = {}, log) {
+    if (log) return {};
```
Comment (wssgcg1213) [inline packages/stylesheet-loader/src/Validation.js]: if (log === false) return {}

File: packages/stylesheet-loader/src/index.js [hunk 4]
```diff
@@ -20,6 +20,10 @@ module.exports = function(source) {
   }
 
   const parsedQuery = loaderUtils.parseQuery(this.query);
+
+  if (typeof parsedQuery.log === 'undefined') {
```
Comment (wssgcg1213) [inline packages/stylesheet-loader/src/index.js]: log = parsedQuery.log === 'true'

File: packages/jsx-compiler/src/modules/css.js (hunk 2 above)
Comment (boiawang) [reply packages/jsx-compiler/src/modules/css.js]: done

File: packages/stylesheet-loader/src/Validation.js (hunk 3 above)
Comment (boiawang) [reply packages/stylesheet-loader/src/Validation.js]: done

File: packages/stylesheet-loader/src/index.js (hunk 4 above)
Comment (boiawang) [reply packages/stylesheet-loader/src/index.js]: done

**Pull request merged**